.gitignore

```diff
@@ -79,5 +79,8 @@ typings/
 # Serverless directories
 .serverless
 
-# Temporary certificate files
-tmp/secrets/*
+# Temporary files
+tmp/
+
+# ignore nat-testing result
```

Comment:
This should go into `tmp/`.

Reply:
I was just running this command manually so I will just use tmp as the output folder from now on :+1: